### ISSUE TITLE
This makes app looks like buggy

### ISSUE THREAD

**Issue by SureshTimma** [fast4x/RiMusic#1241]
### Description

![Screenshot_2024-03-27-11-44-59-955_it fast4x rimusic~2](https://github.com/fast4x/RiMusic/assets/148480722/77f4ceda-9365-47f9-9b83-9afcc13c8a45)

### Suggested Solution

When the songs doesn't have a either have a album or artist the title turns gray and warning icon showing. This looks like very uncomfortable to see.

Please remove that. Instead when clicking on for artist or album it should show there is no album or artist. 

Please

**Comment by sayyid5416** [comment]
Yes I agree. I think this needs to be implemented in some other intuitive way.

**Comment by sayyid5416** [comment]
1. Do not grey out anything. 
2. Show both icon and text normally.
3. On clicking, if album or artist not found - just show a popup that artist/album is not available.

**Comment by SureshTimma** [comment]
> 1. Do not grey out anything. 
> 2. Show both icon and text normally.
> 3. On clicking, if album or artist not found - just show a popup that artist/album is not available.

Yes!!
